architecture/03-data-model.md

```
@@ -0,0 +1,155 @@
+| `role` | enum | `USER` or `ASSISTANT` |
+| `content` | text | Raw message content |
+| `phase` | enum | Which phase this message was sent in |
+| `created_at` | timestamp | |
+
+---
+
+## Git Repository Structure
+
+Each project gets its own Git repository at `git_repo_path`. The repo is initialised when the project is created.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)